lib/asciidoctor.rb

```diff
@@ -501,7 +500,7 @@ class << self; self; end.send :attr_accessor, key
     #                collapsing the line breaks and indentation to
     #                a single space.
     #
-    AttributeEntryRx = /^:(!?\w.*?):(?:#{CG_BLANK}+(.*))?$/
+    AttributeEntryRx = /^:(!?\w.*?):(?:\s+(.*))?$/
```

Comment:
No because that regular expression is only ever run against a single line. Generally speaking, the only regular expressions that run on multiple lines are the ones that have "Inline" in the name (and often with a trailing "m"). It's not a perfect science, but that accounts for most cases.